privacyidea/lib/resolvers/HTTPResolver.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+#
+#  May, 08 2014 Cornelius Kölbel
+#  http://www.privacyidea.org
+#
+#  product:  LinOTP2
+#  module:   httpresolver
+#  tool:     HTTPResolver
+#  edition:  Comunity Edition
+#
+#  Copyright (C) 2010 - 2014 LSE Leading Security Experts GmbH
+#  License:  AGPLv3
+#  contact:  http://www.linotp.org
+#            http://www.lsexperts.de
+#            linotp@lsexperts.de
+#
+# This code is free software; you can redistribute it and/or
+# modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or any later version.
+#
+# This code is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from .UserIdResolver import UserIdResolver
+import requests
+import logging
+import json
+from urllib.parse import urlencode
```

Comment:
Could You use `from six.moves.urllib.parse import urlencode` here to get this to work with Python 2?